### PR TITLE
Comments Q/A

### PR DESCRIPTION
**Pull request by dzucconi**

### CODE REVIEW

**Comment by broskoski** [comment]
Thank ye 👍

File: components/comments/stylesheets/index.styl [hunk 1]
```diff
@@ -1,22 +1,25 @@
...
+
+  &__header
+    //
+
+  &__user
     font-weight bold
-    margin-right (unit/2)
-  > .comment-date-time, > .comment-edited-date-time
+    font-size font-size-sm
+    margin-right 0.5em
```
Comment (broskoski) [inline components/comments/stylesheets/index.styl]: Are we treating `unit` as `1em`?

Comment (dzucconi) [inline components/comments/stylesheets/index.styl]: Yeah, I have been 🤷‍♂️

Comment (broskoski) [inline components/comments/stylesheets/index.styl]: Then I shall as well

**Pull request merged**